File: 01-super-basic-questions/05-check-prime/main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var (
		num     int
		isPrime bool
	)

	fmt.Print("Enter a number above 2: ")
	fmt.Scanf("%d", &num)

	isPrime = true

	for i := 2; i <= int(math.Sqrt(float64(num))); i++ {
		if num%i == 0 {
			isPrime = false
			break
		}
	}

	if isPrime {
		fmt.Printf("The number %d is Prime", num)
	} else {
		fmt.Printf("The number %d is Not Prime", num)
	}
	fmt.Println()

}

File: 01-super-basic-questions/05-check-prime/main.py
import math

num = int(input("Enter a number above 2: "))

isPrime = True

for i in range(2, int(math.sqrt(num))+1):
    if num % i == 0:
        isPrime = False
        break
    
if isPrime:
    print("The number {} is Prime.".format(num))
else:
    print("The number {} is Not Prime.".format(num))